tests/test_translate.py

```diff
@@ -182,3 +182,17 @@ def external_translator(all_objects, output_dir, options):
     expected_keys = set(content.keys()) - set(["$schema"])
 
     assert set(x.name for x in tmpdir.iterdir()) == expected_keys
+
+
+def test_loading_line_number():
+    pings = load_yaml_or_json(ROOT / "data" / "pings.yaml", add_line_number=True)
+    metrics = load_yaml_or_json(ROOT / "data" / "core.yaml", add_line_number=True)
+    invalid = load_yaml_or_json(
+        ROOT / "data" / "invalid_pings.yaml", add_line_number=True
+    )
+    no_line_number_pings = load_yaml_or_json(ROOT / "data" / "pings.yaml")
+
+    assert pings["custom-ping"]["defined_in"]["line"] == "7"
```

Comment:
Is there a reason why the line numbers are strings and not numbers here?  Without a good reason, I think they'd be more natural as numbers (i.e. change `"7"` to `7` here and remove the place where the line numbers are converted to `str` above).

Reply:
This is just a habit from my old Excel days, we always defaulted to using string if we don't plan on doing any calculations with the numbers, mostly to avoid formatting error. So thank you for pointing this out, I have updated it!